src/index.html

```diff
@@ -11,32 +11,40 @@
 </head>
 <body>
     <header class="header">
-            <div class="Leader-header">
-                <h1 class="head">Leaderboard</h1> 
+
+           <div class="Leader-header">
+                <h1 class="head">Leader board</h1> 
             <div>
-            <div class="main-div">
-                <div class="recent_score">
-                    <div class="top_header">
-                        <h2>Recent scores</h2>
-                        <button>Refresh</button>
-                    </div>
-                    <div class="core-holder">
-                        <h3 class="not-colord">Name:100</h3>
-                        <h3 class="colored">Name:10</h3>
-                        <h3 class="not-colord">Name:5</h3>
-                        <h3 class="colored">Name:45</h3>
-                        <h3 class="not-colord">Name:30</h3>
-                        <h3 class="colored">Name:12</h3>
-                        <h3 class="not-colord">Name:130</h3>
-                    </div>
+                <div class="main-div">
+                      <div class="top_header">
+                        <div class="header_box">
+                            <h2>Recent scores</h2> <span></span>
+                            <button class="btn-click">Refresh</button>
+                        </div>
+                        
+                        <div id="user-details">
+                            <div class="my-div"></div>
+                        </div>
+                      </div>
+                      
+                      
+                      <div class="add_your_score">
+                        <h2 class="score-header">Add your score</h2>
+                        <form id="form">
+                            <input type='text' name='user' placeholder="username here" id="user" class="inputs"/>
+                            <input type='integer' name='score' placeholder="Enter Your Score" id="score" class="inputs" />
+                            <button type="submit" class="inputs">Submit</button>
+                          </form>
+                      </div>
+                      
+                      
                 </div>
-                <div class="add_your_score">
-                    <h2 class="score-header">Add your score</h2>
-                    <input class="name" type="text" placeholder="Your name">
-                    <input class="score" type="text" placeholder="Your score">
-                    <input class="submit" type="submit">
-                </div>
-            </div>
+                <div class="error-holder"></div>
+            <!-- <h1> Leader Board</h1> -->
+            
     </header>
+
+  <script src="./index.js"></script>
+  <!-- <script src="./indexAxios.js"></script> -->
```

Comment:
- [ ] To keep your repo clean kindly remove any commented code from your project 🙏